rest-api/alembic/versions/04564bf1b5e8_deidentify_ppi_views.py

```diff
@@ -0,0 +1,279 @@
+   ps.questionnaire_on_overall_health,
+   YEAR(ps.questionnaire_on_overall_health_time) questionnaire_on_overall_health_year,
+   ps.questionnaire_on_lifestyle,
+   YEAR(ps.questionnaire_on_lifestyle_time) questionnaire_on_lifestyle_year,
+   ps.questionnaire_on_the_basics,
+   YEAR(ps.questionnaire_on_the_basics_time) questionnaire_on_the_basics_year,
+   ps.questionnaire_on_healthcare_access,
+   YEAR(ps.questionnaire_on_healthcare_access_time) questionnaire_on_healthcare_access_year,
+   ps.questionnaire_on_medical_history,
+   YEAR(ps.questionnaire_on_medical_history_time) questionnaire_on_medical_history_year,
+   ps.questionnaire_on_medications,
+   YEAR(ps.questionnaire_on_medications_time) questionnaire_on_medications_year,
+   ps.questionnaire_on_family_health,
+   YEAR(ps.questionnaire_on_family_health_time) questionnaire_on_family_health_year,
+   ps.biospecimen_status,
+   YEAR(ps.biospecimen_order_time) biospecimen_order_year,
+   ps.biospecimen_source_site_id,
+   ps.biospecimen_collected_site_id,
+   ps.biospecimen_processed_site_id,
+   ps.biospecimen_finalized_site_id,
+   ps.sample_order_status_1sst8,
+   YEAR(ps.sample_order_status_1sst8_time) sample_order_status_1sst8_year,
+   ps.sample_order_status_1pst8,
+   YEAR(ps.sample_order_status_1pst8_time) sample_order_status_1pst8_year,
+   ps.sample_order_status_1hep4,
+   YEAR(ps.sample_order_status_1hep4_time) sample_order_status_1hep4_year,
+   ps.sample_order_status_1ed04,
+   YEAR(ps.sample_order_status_1ed04_time) sample_order_status_1ed04_year,
+   ps.sample_order_status_1ed10,
+   YEAR(ps.sample_order_status_1ed10_time) sample_order_status_1ed10_year,
+   ps.sample_order_status_2ed10,
+   YEAR(ps.sample_order_status_2ed10_time) sample_order_status_2ed10_year,
+   ps.sample_order_status_1ur10,
+   YEAR(ps.sample_order_status_1ur10_time) sample_order_status_1ur10_year,
+   ps.sample_order_status_1sal,
+   YEAR(ps.sample_order_status_1sal_time) sample_order_status_1sal_year,
+   ps.sample_status_1sst8,
+   YEAR(ps.sample_status_1sst8_time) sample_status_1sst8_year,
+   ps.sample_status_1pst8,
+   YEAR(ps.sample_status_1pst8_time) sample_status_1pst8_year,
+   ps.sample_status_1hep4,
+   YEAR(ps.sample_status_1hep4_time) sample_status_1hep4_year,
+   ps.sample_status_1ed04,
+   YEAR(ps.sample_status_1ed04_time) sample_status_1ed04_year,
+   ps.sample_status_1ed10,
+   YEAR(ps.sample_status_1ed10_time) sample_status_1ed10_year,
+   ps.sample_status_2ed10,
+   YEAR(ps.sample_status_2ed10_time) sample_status_2ed10_year,
+   ps.sample_status_1ur10,
+   YEAR(ps.sample_status_1ur10_time) sample_status_1ur10_year,
+   ps.sample_status_1sal,
+   YEAR(ps.sample_status_1sal_time) sample_status_1sal_year,
+   ps.num_completed_baseline_ppi_modules,
+   ps.num_completed_ppi_modules,
+   ps.num_baseline_samples_arrived,
+   ps.samples_to_isolate_dna,
+   ps.consent_for_cabor,
+   YEAR(ps.consent_for_cabor_time) consent_for_cabor_year,
+   (SELECT IFNULL(GROUP_CONCAT(
```

Comment:
Not sure if we want to do it in this PR, but we may want to consider dropping race_codes and hispanic from this view... they make the view not perform well with lots of participants (e.g. in sandbox.) To do it properly we'll probably have to denormalize things into participant_summary in future, rather than these joins.

Reply:
Followed up with @ssutherland about this. We should keep these columns around for now, even though they don't scale well. We can replace them with a more efficient implementation (reading from new denormalized columns in participant_summary) in future.